### ISSUE TITLE
Can't Add Drops using Dropt

### ISSUE THREAD

**Issue by Spazmunki13** [ferreusveritas/Dynamic-Trees#51]
Hello, I'm trying to make the leaves of dynamic trees drop sticks using Dropt. It works for all leaves except the dynamic leaves and the dynamic leaves from the BOP addon.

I'm not really sure if this is something that I should bring up here or to Dropt.

**Comment by ghost** [comment]
I'll have to take a look at how dropt is doing it.  It's possible that there's a standard I'm not following.

**Comment by ghost** [comment]
How do you have dropt configured? 
 Can you post your config for dropt?

**Comment by Spazmunki13** [comment]
Unfortunately I don`t have the files anymore as I worked around the issue within my modpack.

But if I remember correctly I used CraftTweaker to add the leaves added by Dynamic Trees to an ore dictionary entry and then I added sticks as a drop to any block in that ore dictionary entry.

EDIT: I just realized I closed the issue by accident. If I get the time I can try adding the drops again in the next few days and report back with my config file and the results.
